docusaurus.config.js

```diff
@@ -45,7 +45,13 @@ module.exports = {
         srcDark: "img/logo_white.svg",
         href: "https://statsig.com",
       },
-      items: [],
+      items: [
+        {
+          type: 'html',
+          position: 'right',
+          value: '<button id="consoleCTA" onclick="window.open(\'https://console.statsig.com\', \'_blank\').focus(); window.statsig.logEvent({}, \'console_button_click\', window.location.pathname, {referrer: document && document.referrer,});">Get Started</button>',
```

Comment:
not for this one, let's submit, but we should figure out if this should pass a GA token. Also is referrer: document enough for us to know this came from docs? just curious what type of string is appended here and if we can filter for all docs easily

Reply:
I copied this code from link tracking in docs, confirmed it works but I actually haven't seen the referrer be populated yet. Definitely some TODO here